pkg/acsengine/defaults.go

```diff
@@ -69,10 +69,11 @@ var (
 	}
 
 	//DefaultRHELOSImageConfig is the RHEL Linux distribution.
+	//Being used if no image or distro specified. Default Distro -> RHEL - > this
 	DefaultRHELOSImageConfig = AzureOSImageConfig{
```

Comment:
I would like to keep this as `default`, as its aligns with other code base In this case no input -> RHEL -> DefaultRHELOSImageConfig

Reply:
I don't think we should change this default. Other non-openshift deployments may use it.